apps/tome-api/src/features/card/card.fns.ts

```diff
@@ -750,6 +775,173 @@ export const deck: DbCard[] = [
 			},
 		},
 	},
+	{
+		id: '55',
+		name: 'Burning flame',
+		type: 'spell',
+		colors: ['red'],
+		attack: 18,
+		description:
+			'Every time this spell deals damage, discard the bottom card from its stack. If there’s no more cards to discard, discard Burning flame.',
```

Comment:
I now dont like bottom anymore, somehow I think it should be related to discard green fields?

Reply:
we've never used the "bottom" of a stack, so idk. accepting disagreements though!

Reply:
I was never a fan of "discard the bottom" BUT I was a huge fan at decreasing the stack size and have something happen. So happy for us to remove the mechanic entirely, but I thought that with a little more polishing, this could be cool.

Reply:
yeah I like it "fueling" from something, perhaps we can either use the player’s top of green stack (fuel) rather than bottom of this one? like "whenever this deals damage, discards the top of your green stack, if there isn't any, discard this card"

it really makes it feel like a price to pay. This can also be used for meta reasons (you cast a shit green one, only to discard it afterwards, or something like that)

Reply:
LOVE this. Very nice solution